Oracle/PCORNetLoader_ora.sql

```diff
@@ -936,13 +937,23 @@ insert into pmnencounter(PATID,ENCOUNTERID,admit_date ,ADMIT_TIME ,
 		DISCHARGE_DATE ,DISCHARGE_TIME ,PROVIDERID ,FACILITY_LOCATION  
 		,ENC_TYPE ,FACILITYID ,DISCHARGE_DISPOSITION , 
 		DISCHARGE_STATUS ,DRG ,DRG_TYPE ,ADMITTING_SOURCE) 
-select distinct v.patient_num, v.encounter_num,  
-	start_Date, 
-	to_char(start_Date,'HH:MI'), 
-	end_Date, 
-	to_char(end_Date,'HH:MI'), 
-	providerid,location_zip, 
-(case when pcori_enctype is not null then pcori_enctype else 'UN' end) enc_type, facility_id,  CASE WHEN pcori_enctype='AV' THEN 'NI' ELSE  discharge_disposition END , CASE WHEN pcori_enctype='AV' THEN 'NI' ELSE discharge_status END  , drg.drg, drg_type, CASE WHEN admitting_source IS NULL THEN 'NI' ELSE admitting_source END  
+select 
+  distinct v.patient_num, 
+  v.encounter_num,  
+  start_Date, 
+  to_char(start_Date,'HH:MI'), 
+  end_Date, 
+  to_char(end_Date,'HH:MI'), 
+  '', -- providerid, -- <------------------------------------------------------------------- TODO

```

Comment:
You're commenting out columns that SCILHS sites are required to have. I understand you might have a local branch that comments these out, but please get rid of them from the pull request.